holmes/checks/checks_cli.py

```
@@ -0,0 +1,272 @@
+                console.print(
+                    "[red]No checks file specified and ~/.holmes/checks.yaml not found.[/red]\n"
+                    "Please specify a checks file with --checks-file or use -c for inline checks"
+                )
+                raise typer.Exit(1)
+        # Load checks configuration
+        try:
+            checks_config = load_checks_config(checks_file)
+        except FileNotFoundError:
+            console.print(f"[red]Checks file not found: {checks_file}[/red]")
+            return 1
+        except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)